spec/setup_test_database.rb

```diff
@@ -0,0 +1,5 @@
+require 'pg'
+
+connection = PG.connect(dbname: 'chitter_test')
+
+connection.exec("TRUNCATE peeps;")
```

Comment:
Just incase you weren't aware or ran into any issues later on if you use:
```TRUNCATE peeps RESTART IDENTITY```
It will restart the peep_id values